libs/algorithms/tests/performance/CMakeLists.txt

```diff
@@ -17,6 +17,16 @@ set(benchmarks
     benchmark_unique_copy
     transform_reduce_scaling
    )
+ set(benchmark_inplace_merge_FLAGS DEPENDENCIES hpx_timing)
+ set(benchmark_is_heap_FLAGS DEPENDENCIES hpx_timing)
+ set(benchmark_is_heap_until_FLAGS DEPENDENCIES hpx_timing)
+ set(benchmark_merge_FLAGS DEPENDENCIES hpx_timing)
+ set(benchmark_partition_copy_FLAGS DEPENDENCIES hpx_timing)
+ set(benchmark_partition_FLAGS DEPENDENCIES hpx_timing)
+ set(benchmark_remove_FLAGS DEPENDENCIES hpx_timing)
+ set(benchmark_remove_if_FLAGS DEPENDENCIES hpx_timing)
+ set(benchmark_unique_FLAGS DEPENDENCIES hpx_timing)
+ set(benchmark_unique_copy_FLAGS DEPENDENCIES hpx_timing)
```

Comment:
Hmm, this *should* not be needed just like `config`, `assertion`, `format`, etc. are not needed. Why is this a special case?

Reply:
Right !